Tests/AppTests/API+PackageController+GetRoute+ModelTests.swift

```diff
@@ -119,84 +116,29 @@ class API_PackageController_GetRoute_ModelTests: SnapshotTestCase {
-        let repo = try Repository(
-            package: pkg,
-            forkedFrom: .parentId(originalPkg.requireID()),
-            name: "bar",
-            owner: "foo"
-        )
-        try await repo.save(on: app.db)
-        let version = try App.Version(package: pkg, latest: .defaultBranch, packageName: nil, reference: .branch("main"))
-        try await version.save(on: app.db)
-        
-        let packageResult = try await PackageResult.query(on: app.db, owner: "foo", repository: "bar")
-        let forkedFromResult = try await API.PackageController.GetRoute.fetchForkedFromResult(on: app.db, repository: packageResult.repository)
+    func test_ForkedFromInfo_query() async throws {
+        let originalPkg = try await savePackage(on: app.db, id: .id0, "https://github.com/original/original")
+        try await Repository(package: originalPkg,
+                             name: "original",
+                             owner: "original",
+                             ownerName: "OriginalOwner").save(on: app.db)
+        try await App.Version(package: originalPkg, latest: .defaultBranch, packageName: "OriginalPkg", reference: .branch("main"))
+            .save(on: app.db)
 
         // MUT
-        let model = try XCTUnwrap(API.PackageController.GetRoute.Model(result: packageResult,
-                                                                       history: nil,
-                                                                       products: [],
-                                                                       targets: [],
-                                                                       swiftVersionBuildInfo: nil,
-                                                                       platformBuildInfo: nil,
-                                                                       weightedKeywords: [],
-                                                                       swift6Readiness: nil,
-                                                                       forkedFromResult: forkedFromResult
-                                                                      ))
+        let forkedFrom = await API.PackageController.GetRoute.Model.ForkedFromInfo.query(on: app.db, packageId: .id0)
 
         // validate
-        let info = API.PackageController.GetRoute.Model.ForkedFromInfo.fromSPI(
-            packageName: "bar",
-            originalOwner: "original",
-            originalOwnerName: "OriginalOwner",
-            originalRepo: "original",
-            originalPackageName: "OriginalPkg"
-        )
-        XCTAssertEqual(model.forkedFromInfo, info)
+        XCTAssertEqual(forkedFrom, .fromSPI(originalOwner: "original",
+                                            originalOwnerName: "OriginalOwner",
+                                            originalRepo: "original",
+                                            originalPackageName: "OriginalPkg"))
```

Comment:
The part that we _do_ want to test is the `ForkedFromInfo.query` and so I've converted `test_init_forked_from_spi` into a test for that. It also gets simpler, because we don't need as much set up just to test that query.